netci.groovy

```diff
@@ -63,6 +63,15 @@ platformList.each { platform ->
             osForGHTrigger = "Linux"
             os = "Ubuntu"
         }
+        else if (architecture == 'arm64') {
```

Comment:
It would be nice to not duplicate this block of code for `arm64`.  It seems the only differences are the dockerContainer used and `-DisableCrossgen=true`.

Reply:
I can rework it

Reply:
Done